custom-views-templates/starter-typescript/src/test-utils/index.tsx

```diff
@@ -0,0 +1,103 @@
+import { graphql } from 'msw';
+import { mapResourceAccessToAppliedPermissions } from '@commercetools-frontend/application-shell/test-utils';
+
+export const getDefaultCustomViewServerMocks = () => [
```

Comment:
I would like to move this to the new Custom views test utils file (in the `application-shell` package) but that would imply to include `msw` as a dependency of that package and we don't want that (would be bloating the bundle size).